precompile/contracts/jurorv2/contract.go

```diff
@@ -215,7 +215,7 @@ func getNotionalPositionAndMargin(accessibleState contract.AccessibleState, call
 
 	// CUSTOM CODE STARTS HERE
 	bibliophile := bibliophile.NewBibliophileClient(accessibleState)
-	log.Info("accessibleState.GetSnowContext().ChainID", accessibleState.GetSnowContext().ChainID.String())
+	log.Info("getNotionalPositionAndMargin", "accessibleState.GetSnowContext().ChainID", accessibleState.GetSnowContext().ChainID.String())
```

Comment:
changed cuz it expects vars in pairs after the first var

Reply:
meaning?

Reply:
The expected format is `log.Info("message", "var1", <var1 value>, "var2", <var2 value>, ...)`

So the number of vars is 2n + 1 where n is a whole number. Otherwise it prints the log message but with a warning.